server/views/pages/githubreporequestform.njk

```diff
@@ -0,0 +1,175 @@
+{% extends "../partials/layout.njk" %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %} 
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% block content %}
+<h1 class="govuk-heading-xl">Github repository Request Form</h1>
+
+<h2>After submitting this form, SRE team will run the bootstrap job to create the github repo and service catalog entries. </h2>
+
+{%- set products = [] -%}
+{%- for product in productList -%}
+  {%- set products = (products.push({
+    value: product.value,
+    text: product.text,
+    selected: product.selected
+  }), products) -%}
+{%- endfor -%}
+<form action="/github-repo-request-form" method="POST">
```

Comment:
Again, this needs to be `/form/github-repo-request-form`, the path is made up from `/form` from the `app.ts`, you specify the first part here `app.use('/forms', formsRoutes(services))`, then the second part is inside the routes files itself so `post('/github-repo-request-form', async (req, res): Promise<void> => {`

You're getting a 404 because that URL doesn't exist.